crates/aptos/src/move_tool/mod.rs

```diff
@@ -795,6 +795,11 @@ pub struct RunScript {
     /// Example: `address:0x1 bool:true u8:0`
     #[clap(long, multiple_values = true)]
     pub(crate) args: Vec<ArgWithType>,
+    /// TypeTag arguments separated by spaces.
+    ///
+    /// Example: `u8 u64 u128 bool address vector signer`
```

Comment:
nit, TypeTag might have type tags too. That means, people might write `vector< vector <u8> >` as a single type tag. It is better that you call this out in the comment.

Reply:
Yeah, this is future work